taxcalc/utils.py

```diff
@@ -19,6 +21,7 @@
 except ImportError:
     BOKEH_AVAILABLE = False
 
+TAXCALC_PATH = os.path.dirname(os.path.realpath(__file__))
```

Comment:
I don't know, PR#1022 has been submitted by @zrisher.
Are you saying this line is extraneous?